molecule/commands.py

```diff
@@ -706,7 +706,7 @@ class Init(AbstractCommand):
     Creates the scaffolding for a new role intended for use with molecule.
 
     Usage:
-        init <role> [--docker] [--offline]
+        init <role> [--docker | --openstack] [--offline]
```

Comment:
``Arguments`` and ``Options`` are missing from the docopt docstring.

For instance:
```
  
Arguments:
  <role>  Name of the role

Options:
  --docker      use docker provisioner
  --openstack   use openstack provisioner
  --offline     offline
```